zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -397,8 +399,39 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         self.query_cell_with_type(CellType::LookupByte)
     }
 
-    pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
-        RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // TODO remove me
+    // pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
+    //     RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // }
+
+    // default query_word is 2 limbs
+    pub(crate) fn query_word<const N: usize>(&mut self) -> Word<Cell<F>> {
+        Word::new(
+            self.query_cells(CellType::StoragePhase1, N)
+                .try_into()
+                .unwrap(),
+        )
+    }
+
+    pub(crate) fn query_word4<const N: usize>(&mut self) -> Word4<Cell<F>> {
```

Comment:
Document that the resulting word is not range checked.  (each limb is not guaranteed to be 64 bits)